UI Alterations & APPDATA Reflection

Deadlineem opened the pull request:
Reflected the appdata folder to Terminus instead of HorseMenu
Altered the UI with a more modern design

Also, found a bug with the menu.  It seems you cannot throw any throwable items (Fire bottle, Dynamite, Tomahawk, Bolas, etc.)

![Screenshot_190](https://github.com/user-attachments/assets/4a6a8d2d-e2f4-4f75-8f4f-4367533d9c56)

Deadlineem committed:
UI Restyling for Terminus
Altered APPDATA path from HorseMenu to Terminus

Deadlineem committed:
Update README.md

Updated Screenshot

Deadlineem committed:
Update README.md

Deadlineem committed:
Update README.md

Updated Screenshot

Rxann commented on README.md:
```diff
@@ -15,6 +15,6 @@ Use a popular injector (Xenos/Extreme Injector/Etc.) and inject into rdr2.exe
 ## Screenshots / UI Design
 
-![image](https://github.com/YimMenu/HorseMenu/assets/24372625/e1395e75-7feb-4c4a-9286-bd774e2aaeca)
+![image](https://i.ibb.co/WvTJrLcM/Screenshot-190.png)
```
Reasoning for changing the hosting? I think having it here is better

Deadlineem replied:
I simply didn't have the image uploaded to github/PR made yet is all.  ImgBB has no expiration for this one so i figured it was fine lol

Rxann replied:
Alright, that is fine til now. Don't resolve it until it's hosted on our github assets

Rxann reviewed (commented):
Also, I think that we should have some sort of file migration logic so the files don't get all reset when the location is changed, for example I have a large player DB that I wouldn't want to have to copy manually.

Rxann commented:
Could you send your settings file for analysis to make sure something else isn't at fault?

Deadlineem commented:
> Could you send your settings file for analysis to make sure something else isn't at fault?

[settings.json](https://github.com/user-attachments/files/18678593/settings.json)
%APPDATA%\Terminus

was a fresh folder, i just did a reinstall on my pc a few days ago, too so horsemenu folder isnt in appdata, just terminus.

Deadlineem commented:
> Also, I think that we should have some sort of file migration logic so the files don't get all reset when the location is changed, for example I have a large player DB that I wouldn't want to have to copy manually.

This is a good idea, should we just implement a check to see if the HorseMenu folder exists in APPDATA and just copy the files from that into Terminus folder?  We could optionally leave the HorseMenu folder in APPDATA as well, or have it delete it after the files are copied.

Maybe an onboarding screen like YimMenu has with options to select if you would like to scan for horsemenu and merge your old data to terminus?

Rxann commented:
> > Also, I think that we should have some sort of file migration logic so the files don't get all reset when the location is changed, for example I have a large player DB that I wouldn't want to have to copy manually.
> 
> 
> 
> This is a good idea, should we just implement a check to see if the HorseMenu folder exists in APPDATA and just copy the files from that into Terminus folder?  We could optionally leave the HorseMenu folder in APPDATA as well, or have it delete it after the files are copied.
> 
> 
> 
> Maybe an onboarding screen like YimMenu has with options to select if you would like to scan for horsemenu and merge your old data to terminus?

Yeah I would say something like this. I will probably develop this feature myself, but I just wanted to get opinions on here.

MarkEcza commented on src/main.cpp:
```diff
@@ -21,7 +21,7 @@ namespace YimMenu
 	static DWORD Main(void*)
 	{
-		const auto documents = std::filesystem::path(std::getenv("appdata")) / "HorseMenu";
+		const auto documents = std::filesystem::path(std::getenv("appdata")) / "Terminus";
```
Kinda misleading having it named documents and it being in appdata, i think we should actually move this to documents.

Rxann replied:
This would make sense. I am fine with appdata but if we are going to migrate anyway we might aswell move it now.

Deadlineem replied:
> Kinda misleading having it named documents and it being in appdata, i think we should actually move this to documents.

I think this would be a good idea, would make it easier for people to locate the Terminus folder

Deadlineem committed:
Features, Rewards, Fixes & Improvements

Deadlineem committed:
Added Single Player Mods
- Add Money
- Remove Fog of War from Map
- Give All Satchel Items (should work in SP/MP)
Added ItemTypes (For Satchel)
Added more WeaponTypes to cover missing weapon entries

Deadlineem committed:
Added Min/Max Honor for Single Player
Added a train speed native for testing (no change)
Added a Story subcategory to the 'Recovery' category
